index.js

```diff
@@ -27,8 +27,8 @@ function writeConfigToDisk (platform, config, v1Config) {
         configName = platform.replace(BROWSERS_SUBDIR, 'extension-')
     }
 
-    fs.writeFileSync(`${GENERATED_DIR}/v2/${configName}-config.json`, JSON.stringify(config, null, 4))
-    fs.writeFileSync(`${GENERATED_DIR}/v1/${configName}-config.json`, JSON.stringify(v1Config, null, 4))
+    fs.writeFileSync(`${GENERATED_DIR}/v2/${configName}-config.json`, JSON.stringify(fixReasons(config), null, 4))
+    fs.writeFileSync(`${GENERATED_DIR}/v1/${configName}-config.json`, JSON.stringify(fixReasons(v1Config), null, 4))
```

Comment:
I thought about it a bit more and I think we should change this code:

- having `fixReasons` in `writeConfigToDisk` is kinda unexpected (name suggests that we are only writing to disk), also, all other processing happens before this function. Let's move `fixReasons` to before `writeConfigToDisk` is called. If you call it before v1 config is generated you can probably get away with calling it once instead of twice per platform.
- I'd change the name from `fixRasons` to something more explicit e.g. `inlineReasonArrays`